_lessons/who-is-analytics-engineer.html

```diff
@@ -27,17 +27,24 @@
 ]
 ]
 
+???
+Slide setup: When did people first start hearing the term 'analytics engineering'? Analytics Engineering is relatively new. The role of Analytics Engineer is relatively new. To understand it, we need to understand how data teams traditionally looked. 
```

Comment:
praise: Nice teacher moves here! you are giving excellent context for why we are talking about traditional data teams